tools/deployment/package-helm/templates/shared-data-archives-pv.yaml

```
@@ -0,0 +1,9 @@
+{{- include "clp.createLocalPv" (dict
+  "root" .
+  "component_category" "shared-data"
+  "name" "archives"
+  "nodeRole" "control-plane"
+  "capacity" "50Gi"
+  "accessModes" (list "ReadWriteMany")
+  "hostPath" (printf "%s/archives" .Values.clpConfig.data_directory)
```

```suggestion
  "hostPath" .Values.clpConfig.archive_output.storage.directory
```

Unless I'm missing something?